### ISSUE TITLE
Suggestion: Solo Nodifications

### ISSUE THREAD

**Issue by MichaelTheAvali** [ipodtouch0218/NSMB-MarioVsLuigi#136]
I tend to play solo when no one joins my lobby, but I always have to keep ending the match early just to check if someone's joined, so maybe make it so it alerts you when your playing solo if someone joins the lobby so I don't have to constantly check

**Comment by SLG64** [comment]
I like this idea. It could say "(PlayerUsername) joined the party."

**Comment by ipodtouch0218** [comment]
Are you talking about actively playing a game, but with only 1 player in the lobby? Like not in the room menu? A "player joined the room" and "player left the room" feature has been planned for a while, just haven't gotten a chance to start on it yet.

> @SLG64 And maybe the sound is a simple jingle. I got a couple:
Those are files on your machine. We can't see them.

**Comment by SLG64** [comment]
Oops, my bad.

**Comment by MichaelTheAvali** [comment]
> Are you talking about actively playing a game, but with only 1 player in the lobby? Like not in the room menu? A "player joined the room" and "player left the room" feature has been planned for a while, just haven't gotten a chance to start on it yet.

Yeah, its just cause I like to play the game solo at times instead of sitting and waiting at the lobby screen for someone to join

**Comment by drok28** [comment]
Yeah, I do that too.

**Comment by SLG64** [comment]
What I was trying to put in was an unused jingle for SM3DL, which I got from the TCRF.

**Comment by ipodtouch0218** [comment]
> I just realized I can just send a link to the file. My bad! Anyways here it is:
> 
> https://tcrf.net/images/transcoded/3/3c/Sm3dl_unused_mario_jingle.ogg/Sm3dl_unused_mario_jingle.ogg.mp3

Doesn't really fit as a "player joining sound" imo. It'd probably be best to just use the existing "player join" and "player leave" sounds from the minigame mode. Mostly because they already have the same meaning.

**Comment by SLG64** [comment]
Oh, that's alright.
